src/reporter.js

```diff
@@ -1,20 +1,37 @@
 import request from "browser-request";
 
-const url = "https://beta.accesslint.com/api/v1/reports";
+const url = "/access_lint/errors"
 
 export default function (message) {
-  const violations = message.violations;
+  var violations = message.violations.map(function(violation) {
+    return {
+      description: violation.description,
+      help: violation.help,
+      helpUrl: violation.helpUrl,
+      id: violation.id,
+      nodes: violation.nodes.map(function(n) {
+        return {
+          target: n.target,
+          html: n.html,
+          impact: n.impact
+        };
+      }),
+      tags: violation.tags
+    };
+  });
 
   if (violations.length > 0) {
     request({
       method: "POST",
       url: url,
-      json: message,
+      json: {
+        accesslint: {
+          violations: violations,
+          url: window.location.pathname
+        }
+      }
     }, function() {});
-
-    console.error(
-      `AccessLint - ${violations.length} accessibility violations:`,
-      violations
-    );
   }
+
+  console.log("AccessLint warnings: ", violations);
```

Comment:
If it does, I think in ESLint you can define available globals, so you can say `console` is there and Hound won't annoy you about it any longer.